Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:alpine 
+COPY ./ /opt 
+WORKDIR "/opt" 
+RUN npm cache clean --force && rm -rf node_modules && npm install 
```

Comment:
We don't need to clean cache in a docker image

Reply:
@subhatanay can you please look into this?